samples/snippets/cloud-client/src/main/java/recaptcha/account_defender/AccountDefenderAssessment.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package account_defender;
+
+// [START recaptcha_enterprise_account_defender_assessment]
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.protobuf.ByteString;
+import com.google.recaptchaenterprise.v1.AccountDefenderAssessment.AccountDefenderLabel;
+import com.google.recaptchaenterprise.v1.Assessment;
+import com.google.recaptchaenterprise.v1.CreateAssessmentRequest;
+import com.google.recaptchaenterprise.v1.Event;
+import com.google.recaptchaenterprise.v1.ProjectName;
+import com.google.recaptchaenterprise.v1.RiskAnalysis.ClassificationReason;
+import com.google.recaptchaenterprise.v1.TokenProperties;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+import java.util.UUID;
+
+public class AccountDefenderAssessment {
+
+  public static void main(String[] args) throws IOException, NoSuchAlgorithmException {
+    // TODO(developer): Replace these variables before running the sample.
+    // projectId: Google Cloud Project ID
+    String projectId = "project-id";
+
+    // recaptchaSiteKey: Site key obtained by registering a domain/app to use recaptcha
+    // services.
+    String recaptchaSiteKey = "recaptcha-site-key";
+
+    // token: The token obtained from the client on passing the recaptchaSiteKey.
+    // To get the token, integrate the recaptchaSiteKey with frontend. See,
+    // https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+    String token = "recaptcha-token";
+
+    // recaptchaAction: The action name corresponding to the token.
+    String recaptchaAction = "recaptcha-action";
+
+    // Unique ID of the customer, such as email, customer ID, etc.
+    String uniqueCustomerId = "default" + UUID.randomUUID().toString().split("-")[0];
```

Comment:
Did you consider to align the variable names to the documentation?

https://cloud.google.com/recaptcha-enterprise/docs/account-defender#interpret-assessment-details uses "user identifier", not "customer ID".

Reply:
Thanks for pointing it out! Changed the var name. PTAL